### PR TITLE
Feature/main event squadscore

### PR DESCRIPTION
**Pull request by VPDeb**
Created Main.py for Squad Score for Classroom Rumble app.

### CODE REVIEW

File: scripts/process_submission/main.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+def decode(x):
+    return json.JSONDecoder().decode(x)
+def encode(x):
+    return json.JSONEncoder().encode(x)
+input = decode(sys.argv[1])
+
+log = logging.getLogger(__name__)
+vision = GoogleAPI()
+{maybe boto3}
```
Comment (lorischl-otter) [inline scripts/process_submission/main.py:20]: This line should be removed -- it was just part of my transfer notes :)

File: scripts/process_submission/main.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+s3 = boto3.client(
+    service_name='s3',
+    region_name=getenv("S3_REGION"),
+    aws_access_key_id=getenv("AWS_ACCESS_KEY_ID"),
+    aws_secret_access_key=getenv("AWS_SECRET_ACCESS_KEY")
+)
+async def submission_text(sub: Submission):
+    transcriptions = ""
+    confidence_flags = []
+    for page_num in sub.Pages: (maybe not for loop if only 1 page):
```
Comment (lorischl-otter) [inline scripts/process_submission/main.py:31]: This parenthetical `(maybe not...)` should also be removed. @bramirez96 can you confirm whether Classroom Rumble can have multiple page submissions? Or just single page? If just single page, we could remove this from a for loop if we want to.

File: scripts/process_submission/main.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                # fetch file from s3 bucket
```
Comment (lorischl-otter) [inline scripts/process_submission/main.py:32]: Looks like there are some remaining tabbing issues from my Slack message -- looks like the code might be alright, but a few of the comments here look a little misaligned.

File: scripts/process_submission/main.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        r = s3.get_object(Bucket=getenv("S3_BUCKET"), Key=filekey)
+        r_content = r['Body'].read()
+        conf_flag, flagged, trans, rotation = await vision.transcribe(r_content)
+       # concat transcriptions together
+        transcriptions += trans + "\n"
+        # add page to list of confidence flags
+        confidence_flags.append(conf_flag)
+   # score the transcription using SquadScore algorithm
+    score = await squad_score(transcriptions, scaler)
+    # change this ??
```
Comment (lorischl-otter) [inline scripts/process_submission/main.py:44]: This comment can be removed

**Review by lorischl-otter — CHANGES_REQUESTED** [review]
Thanks for getting this uploaded. I added in a few change requests. 

Also, looks like you're already on top of this for the next push, but if you can get rid of the .ipynb_checkpoints & the second `fdbkassign` notebook, that would be great. Also, unless @bramirez96 has a different preference, if you can keep the changes on these files in one PR, that would be helpful to track what you're changing each time. You can just push any new changes from this PR to the same branch, and it will update here. 

Let me know if you have any Qs!

File: scripts/process_submission/main.py (hunk 2 above)
Comment (bramirez96) [reply scripts/process_submission/main.py:31]: Just single page for now!

**Comment by bramirez96** [comment]
I agree, clean the code up, and let's get it all into one PR. There are currently 3 open and I think they're all your branches.

**Comment by bramirez96** [comment]
Also, make SURE to switch the branch from `main` to `development`. Click on `Edit` next to your PR title to change it.

**Pull request closed without merging**